src/components/scheduling/SchedulingEditor.svelte

```diff
@@ -30,7 +30,7 @@
 
 <Panel overflowYBody="hidden">
   <svelte:fragment slot="header">
-    <Chip>{title}</Chip>
+    <SectionTitle>{title}</SectionTitle>
```

Comment:
In the future for changes like this which are throughout, and do not seem specific to this PR we should just do a quick separate commit/PR. Will help keep reviewing more targeted.

Reply:
Yeah agreed, had that thought a little while after this change. Will split things like this out in future PRs.